install_config/topics/gce_scopes_permissions-con.adoc

```diff
@@ -0,0 +1,75 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+= Configuring authorization for Google Compute Engine
+
+*Permissions*
+
+Configuring Google Compute Engine (GCE) for {product-title} requires the following role:
+
+[cols='1,3']
+|===
+
+|`roles/owner`
+|Needed for creating service accounts, cloud storage, instances, images,
+templates, Cloud DNS entries, and to deploy load balancers and health checks. It
+is helpful to also have `delete` permissions to be able to redeploy the
+environment while testing.
+|===
+
+For more information on GCE roles, link:https://cloud.google.com/iam/docs/understanding-roles[see the Understanding roles section of the GCE documentation].
+
```

Comment:
Maybe we can add here:

It is recommended to create a service account to avoid using personal users when deploying Google Cloud Platform objects.